### DISCUSSION TITLE
最新版本终于来了. 7.8.5 请各位网络大神来下载! 或者有些用户已自动更新了! 请查看!

### DISCUSSION THREAD

**Discussion by MaxMKMax** [getlantern/lantern-forum-cn#383] [Announcements]
各位好! 这两个链接都可以下载. 这个版本 7.8.4 应该修复了闪退和别的问题了. 如果还有什么问题, 请再来反馈.
谢谢你们耐心的等待! 

https://github.com/getlantern/lantern/blob/devel/README.zh.md

https://github.com/getlantern/lantern

**Comment by set-aua** [comment]
谢谢 🙏 Mike

**Reply by MaxMKMax** [reply]
Hi set-aua, 不客气啊! 谢谢您支持蓝灯! 🙏 给你们最好的服务和app是我们的初心和最终目标!

**Reply by jean-reno** [reply]
要是实现自选服务器 就好了

**Reply by 888sssjjjjgfyh** [reply]
我的专业版删除了文件，现在是普通版了，怎么恢复正常
wangxiaoguang1690@gmail.com

**Reply by MaxMKMax** [reply]
@888sssjjjjgfyh 您好. 我们会帮您升级或发激活码给您. 在GitHub或邮件. 请留意.

**Reply by Jumpsteelrail** [reply]
我也是专业版删除了文件，现在还是限流量的普通版，麻烦您帮我解决一下可以吗？
jumpsteelrail@126.com
谢谢麦克

**Reply by cc-endyx** [reply]
qq577276939@gmail.com  我也是专业版删除了文件，现在还是限流量的普通版，麻烦您帮我解决一下可以吗？
![微信图片_20240405191414](https://github.com/getlantern/lantern-forum-cn/assets/166127417/c5c40ac3-2e40-44c2-ac05-40e1a95b6541)

**Reply by MaxMKMax** [reply]
@cc-endyx 您好. 我给您发激活码了. 请您查看邮箱. 标题: 蓝灯激活码

**Reply by link111111111** [reply]
我也是同样的问题 可以帮我解决一下么 83947926@qq.com

**Reply by YAHAHA02** [reply]
@Mike777Khoo 激活码请发邮箱 gj_lavender@hotmail.com

**Reply by YayuZhou123** [reply]
@Mike777Khoo 我也是同样的问题。激活码请发邮箱 330266506@qq.com

**Reply by txwgoogol** [reply]
@Mike777Khoo 我也遇到同样的问题，激活码请发邮箱colin.googol@gmail.com

**Comment by frankvaliant** [comment]
macos还是7.8.3 啊

**Reply by wgyulin** [reply]
MAC OS 怎么还是7.8.3版本呀

**Reply by MaxMKMax** [reply]
@frankvaliant @wgyulin 您好. 请您确定看是否有更新到最新版本呢. 7.8.4 的.

**Comment by EFLql** [comment]
win11实测还是7.8.3

**Reply by kuukuuukuuu** [reply]
感觉像耍猴

**Reply by MaxMKMax** [reply]
@EFLql 您好. 请您确定看是否有更新到最新版本呢. 7.8.4 的.

**Reply by EFLql** [reply]
原来7.8.4是在稳定版win10+的那里的第二个下载链接，第一个下载链接还是7.8.3没有更新

**Comment by carpediemcc** [comment]
老哥，更新完确实是不闪退了，但是还是显示连上了但是啥都打不开啊。这眼瞅着快一个月了，排查一大圈了，就差换网络运营商了，能解决一下不，我看别人也发过类似的问题
![Uploading 1.png…]()

**Comment by YAHAHA02** [comment]
@Mike777Khoo 现在的问题是如何获取自己的激活码？邮箱gj_lavender@hotmail.com

**Reply by MaxMKMax** [reply]
@YAHAHA02 您好! 不好意思, 激活码哪里暂时出问题. 需要手动发送. 我来帮您找客服要. 
您这两天里查看邮箱.

**Comment by dna0606** [comment]
@Mike777Khoo，更新了，还是老样子，点不开闪退，我要写论文不想耽误时间，请问可以退费吗

**Reply by MaxMKMax** [reply]
@dna0606  您好! 照道理应该不会闪退了. 因为已经修好了, 其他人也没问题了. 您确定是最新版本吗? 7.8.4

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队.

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Comment by cai6685** [comment]
@Mike777Khoo，为什么我下载安装你发的链接后还是显示7.8.3版本。

**Reply by MaxMKMax** [reply]
@cai6685 看来少数的用户会遇到像您这样的问题. 也不知道为什么.

我刚才帮您问了, 看技术团队怎么解决这个问题. 谢谢您的反馈.

您试试看重新下载呢?

**Comment by taoistx** [comment]
是没有闪退了，但之前删掉global.yami登录不上专业版了，怎么版？是支付宝购买的，没有激活码

**Reply by taoistx** [reply]
@Mike777Khoo

**Comment by taoistx** [comment]
PC版的下载后怎么知道是什么版本？界面没有显示任何版本号，下载链接里日志也只是写到7.82

**Reply by taoistx** [reply]
@Mike777Khoo

**Reply by MaxMKMax** [reply]
@taoistx 他们没有更新页面吧. 您下载后就是最新版本了. 在设定那里可以看到.

**Reply by 396738631** [reply]
pc 版的win7的一直会闪退，在最新面页下载后，在卸载程序里面看版本到更新后的还是7.8.2啊，win10的可以用啊，请给个稳定可下载的版本吧

**Comment by fireopenwahaha** [comment]
@Mike777Khoo 你好，我注册付费用户时填错了邮箱，现在收不到激活码，我已经向蓝灯发送了多封邮件，提供了付费信息，但是都没有给我解决，我的邮箱是atie0@163.com，如果你能看到这条信息，请查阅我的邮件，希望能够尽快解决问题。

**Reply by MaxMKMax** [reply]
@fireopenwahaha 您好! 我在另一个帖子回复您了. 我们会帮您升级或发激活码给您. 在GitHub或邮件. 请留意.

**Comment by jean-reno** [comment]
我怎么还是7.8.1 也没自动更新 你们说的7.8.4 在哪里？

**Comment by xiaoigl** [comment]
我下载了beta版好像是7.8.5 请问怎么用啊 之前也是7.8.1 闪退

**Reply by MaxMKMax** [reply]
@xiaoigl 你好。你可以参考这里：
https://lantern.io/zh-Hans/faq

**Comment by Faith0823** [comment]
7.8.4仍然没有修复上网的问题，依然上不去google都上不去。用了6年，这次感觉不会再续费了啊

**Reply by MaxMKMax** [reply]
@Faith0823 对于造成的不便，我们深表歉意。他们现在正在解决问题.

**Reply by beta-zhou** [reply]
之前解决了账号不能登录的问题，现在还是一直在闪退啊，用都不给用了吗？

**Reply by MaxMKMax** [reply]
@beta-zhou 您好. 请您检查看是否更新到了最新版本? version 7.8.4 . 闪退问题应该已经解决了.

**Comment by MaxMKMax** [comment]
@888sssjjjjgfyh 我们的客户服务团队表示您可以尝试电子邮件链接。您应该能够接收授权码，特别是因为您使用的是 gmail.

**Comment by songlee50** [comment]
没有自动更新，下载安装包安装并重启后依然显示为7.8.3 (20240305.214140)，网站连接依旧困难，无法打开openai和copilot。讽刺的是，不开的情况下还可以访问github，开了以后连这个问题都无法访问和回复。

**Reply by songlee50** [reply]
另注：下载安装的是稳定版，无法下载测试版。

**Reply by songlee50** [reply]
已下载测试版安装，看起来问题暂时解决了。

**Reply by MaxMKMax** [reply]
@songlee50 
您好. 有时候 openAI 用不了是因为他们侦测我们为机器人之类的问题导致的. 这说起来挺复杂的. 但我们的技术团队已经非常努力的去绕开侦测和解决问题了.

**Reply by MaxMKMax** [reply]
@songlee50 
嗯嗯. 那就好. 谢谢您支持蓝灯!

**Comment by cc-endyx** [comment]
[image: 1712405426379.jpg]并没有用

songlee50 ***@***.***> 于2024年4月6日周六 17:10写道：

**Reply by MaxMKMax** [reply]
@cc-endyx 您好! 我这里看不到您发的图像和您写的邮箱. 

您可以分开写您的邮箱. 比如: s u p p ort @ g e t lan tern. o r g

看来被GitHub屏蔽了. 请问您需要什么帮忙吗?

**Reply by cc-endyx** [reply]
qq 577276939 @ g m a i l . c o m
![Uploading 1712411023808.jpg…]()

**Reply by MaxMKMax** [reply]
@cc-endyx 
好的. 我会帮您提交问题和获取激活码. 过几天请查看邮箱和GitHub. 谢谢您耐心的等待和支持蓝灯!

**Comment by Fulda1985** [comment]


**Reply by Fulda1985** [reply]
@Mike777Khoo

**Reply by MaxMKMax** [reply]
@Fulda1985 好的. 我会帮您提交问题和获取激活码. 也请提供我付款单据. 谢谢!

过几天请查看邮箱和GitHub. 谢谢您耐心的等待和支持蓝灯!

**Reply by Fulda1985** [reply]
付款单据：
商户：Wellnity, LLC
订单金额：340元
支付方式：支付宝
时间：2023-10-30 19:37:56
订单号：2023103022001337241414698488
商家订单号：src_1O6tlXDTyZkPGbMkVxnhaRIR
@Mike777Khoo

**Reply by Fulda1985** [reply]
单据已经提供，能否帮忙申请发送激活码到邮箱，谢谢！
@Mike777Khoo

**Reply by MaxMKMax** [reply]
@Fulda1985 好的. 我帮您提交上去. 周末团队不在线, 要等星期一晚上了. 谢谢您的等待.

**Reply by Fulda1985** [reply]
@Mike777Khoo 还是没收到回复呢。

**Comment by sia08051005** [comment]
您好，当时购买专业版使用的邮箱是siyanpku@163.com，但是每次都是购买一年的，找不到当时的付款单据了，而且当时是直接购买没有邀请码的，请问现在要怎么解决恢复不到专业版的问题？谢谢！另外您回复消息可以发送到siyanpku@gmail.com

@Mike777Khoo

**Reply by MaxMKMax** [reply]
@sia08051005 好的. 我会帮您提交问题和获取激活码. 

如果没有付款单据应该也没关系. 我让他们看系统里有记录吗.

过几天请查看邮箱和GitHub. 谢谢您耐心的等待和支持蓝灯!

**Reply by sia08051005** [reply]
你好，这边依然没有收到激活码，请您尽快处理帮忙恢复到专业版，谢谢 @Mike777Khoo

**Reply by sia08051005** [reply]
您好，之前一直都是免费版，更新了7.8.5之后蓝灯依然是免费版，您们的工作人员也没有给我发过激活码，请尽快处理！！！@Mike777Khoo

**Comment by Qeedo082455** [comment]
现在退回到 v 7.7.0版本，PC端测试可以用。只是希望不要自动升级到7.8.1版本。

**Reply by MaxMKMax** [reply]
@Qeedo082455 请问您是用哪个系统? IOS? MacOS? Window?

**Reply by Qeedo082455** [reply]
我是Windows的系统，v 7.8.4的安装包没找到，7.8.1和7.8.3（只能安装后在控制面板里面才能看到版本号）试过都不行。好在之前电脑里面还保留有去年12月下载的7.7.0安装包，试了一下反而可以连上认证服务器（有时候会卡，但能用），目前为止都还算正常。

建议新版本安装包放出的时候，直接在名称上标注版本，不然名字都一样分不清。

**Comment by YAHAHA02** [comment]
@Mike777Khoo 依然还是没有收到激活码 请尽快查看发送以下 谢谢 邮箱 gj_lavender@hotmail.com

**Reply by MaxMKMax** [reply]
@YAHAHA02 您好. 请您也提供我付款单据. 我来帮您提交问题. 请您耐心等待, 可能需要 2 天来帮您手动升级完成. 谢谢!

**Comment by jywxsm** [comment]
什么时候给我们激活码啊，一直是免费版。用不了专业版。

**Reply by MaxMKMax** [reply]
@jywxsm 请您提供我账号邮箱和付款单据. 谢谢.
同时您可以试试看重新登录. 看可以吗. 记得下载最新版本.

**Comment by songlee50** [comment]
请把我从邮件列表中删除，不要抄送给我，谢谢！




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年4月9日(星期二) 晚上8:47
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [getlantern/lantern-forum-cn] 最新版本终于来了. 7.8.4 请各位网络大神来下载! 或者有些用户已自动更新了! 请查看! (Discussion #383)














 
什么时候给我们激活码啊，一直是免费版。用不了专业版。

**Reply by MaxMKMax** [reply]
@songlee50 好的. 没问题. 您也可以在邮箱取消关注.

**Comment by xinleiye** [comment]


**Reply by MaxMKMax** [reply]
@xinleiye 嗯嗯好的. 我会帮您提交给付款部门. 请您耐心等待, 可能需要 2 天来帮您手动升级完成. 谢谢!

同时您可以试试看重新登录. 看可以吗. 记得下载最新版本.

**Reply by MaxMKMax** [reply]
@xinleiye 请您也提供我付款单据. 我来帮您提交问题. 谢谢!

**Comment by mqingqing123** [comment]
不会升级的可以查看此处  https://github.com/getlantern/lantern-forum-cn/discussions/421

**Comment by 407698432** [comment]
wifi还是用不上，能不能解决啊

**Reply by MaxMKMax** [reply]
@407698432 技术团队说事您的网络服务供应商的问题. 他们一直更新阻碍翻墙. 所以很难解决了

**Reply by 407698432** [reply]
怎么可能，我现在用快连vpn就能用，就你这个不行

**Reply by 407698432** [reply]
那我可以申请退款吗

**Reply by MaxMKMax** [reply]
@407698432 您好, 请问您可以向我们的支付团队发送电子邮件吗？因为我们想保护您的个人信息和隐私。因此，最好直接向我们的支付团队发送电子邮件申请退款。谢谢。

电子邮件：support@getlantern.org

**Comment by taoistx** [comment]
***@***.*** 支付宝购买专业版问题 · getlantern/lantern-forum-cn · Discussion #394 (github.com) 在这个贴子里反馈过情况，现在一周多时间了，目前有什么进展吗？

**Comment by 396738631** [comment]
> 各位好! 这两个链接都可以下载. 这个版本 7.8.4 应该修复了闪退和别的问题了. 如果还有什么问题, 请再来反馈. 谢谢你们耐心的等待!
> 
> https://github.com/getlantern/lantern/blob/devel/README.zh.md
> 
> https://github.com/getlantern/lantern

win7版本的下载了还是7.8.2，安装了之后显示的版本也是7.8.2

**Reply by MaxMKMax** [reply]
@396738631 您好. 请您试试看这个链接:

Mac - https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-preview.dmg

Windows - https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-preview.exe

Linux - https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-preview-64-bit.deb

**Reply by ycjdixon** [reply]
这个链接下载的安装文件，win7系统安装后打不开。现在蓝灯是不想管WIN7用户了吗？

**Reply by MaxMKMax** [reply]
@ycjdixon 
您好. 我们怎么可能会不管window的用户呢? 一会我和他们开会, 我来反馈给技术团队.
非常抱歉让您久等了. 谢谢您对我们的不离不弃. 谢谢您支持蓝灯!

**Comment by ycjdixon** [comment]
最新版本终于来了. 7.8.5.为什么每一次升级都轮不到WIN7版本？

**Reply by MaxMKMax** [reply]
@ycjdixon 您好, 那天我又和技术团队开会了. 我们已经把window 7 的需求升级到紧急需要了. 他们会关注了!
不好意思让您久等了, 应为之前没想到 Window 7 这么多人有需求.

**Reply by woodchuckk** [reply]
什么时候能好，现在还是断断续续的用着旧版，真的心累

**Reply by MaxMKMax** [reply]
@woodchuckk 您好. 您用的WiFi是哪家的? 我们觉得网络供应商应该是有影响到用户体验. 因为有些完全顺畅, 有些用户向您一直遇到问题. 所以我们想搞清楚您使用哪家的网络?

**Reply by woodchuckk** [reply]
是电信，但是跟这个没关系啊…最早的几年用起来非常丝滑…自从改版了之后感觉win7就被抛弃了一样，你们的团队应该是后接手的吧，之前的团队是不是都走了。

**Reply by 396738631** [reply]
windows7 什么时候会更新了，注册了两年费用有两个多月不能用了。

**Comment by peisy** [comment]
为何还没有ios新版本？

**Reply by MaxMKMax** [reply]
@peisy 已经在筹备着了, 应该两周内到一个月内有机会可以有最新版本了.

**Reply by peisy** [reply]
@Mike777Khoo ios新版本筹备的好了嘛？马上又一个月了

**Reply by MaxMKMax** [reply]
@peisy 您好. 很抱歉还没好呢. 卡在某些地方所以还没弄好. 我再帮你们催一下. 不好意思让你们久等了.

**Reply by peisy** [reply]
@Mike777Khoo ios新版本筹备的好了嘛？马上又又一个月了

**Reply by tytechan** [reply]
给我看乐了。。他们老这么承诺的解决问题到最后就石沉大海了

**Comment by Wwww1237** [comment]
7.8.5是最新的吗？我的蓝灯页面显示的还是7.8.4，而且最近不是很稳定，掉线频率有点高（win 10；surface）

**Comment by 396738631** [comment]
都试过了，关键下载界面lantern/README.zh.md at devel · getlantern/lantern (github.com)&nbsp;稳定版 根本就没有更新win7系统下的安装包，安装后还是闪退，去查看下载的版本依然是7.8.2，你们要考虑一下老电脑用户的需求啊，都已经一个多月不能使用了。


恋曲1980
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年4月15日(星期一) 晚上11:51
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 最新版本终于来了. 7.8.4 请各位网络大神来下载! 或者有些用户已自动更新了! 请查看! (Discussion #383)





 
@396738631 您好. 请您试试看这个链接:
 
Mac - https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-preview.dmg
 
Windows - https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-preview.exe
 
Linux - https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-preview-64-bit.deb

**Comment by 396738631** [comment]
申请退款，账号 ***@***.***

专业版02/06/2025到期，已经半年无法正常使用Win7，你们一直不能修正。



---原始邮件---
发件人: "Mike ***@***.***&gt;
发送时间: 2024年5月15日(周三) 凌晨1:00
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [getlantern/lantern-forum-cn] 最新版本终于来了. 7.8.5 请各位网络大神来下载! 或者有些用户已自动更新了! 请查看! (Discussion #383)




 
@peisy 您好. 很抱歉还没好呢. 卡在某些地方所以还没弄好. 我再帮你们催一下. 不好意思让你们久等了.
